webpack.circleci.config.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview CircleCI e2e tests environment config file for Webpack.
+ */
+
+const merge = require('webpack-merge');
+const common = require('./webpack.common.config.ts');
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = merge(common, {
```

Comment:
Let's inherit from the prod config here and change just the Terser configuration.

Reply:
Done!